h3/src/proto/frame.rs

```diff
@@ -127,6 +184,66 @@ impl fmt::Display for Frame {
         }
     }
 }
+
+impl fmt::Debug for Frame<PayloadLen> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        <Self as fmt::Display>::fmt(self, f)
+    }
+}
+
+impl<B> fmt::Display for Frame<B>
```

Comment:
I realize this was here before, so feel free to ignore, but: is there a need for `Display` for this type? It's an internal type, and the only fmt purpose is for logs, right? Since `Display` is often meant for "turn this type into a value you'd show a user", perhaps `Debug` is all that is needed.

Reply:
Done in 2bb75d6.

Don't know why I implemented `Display` :sweat_smile: .